features/checkout/placing_order_on_multi_channel/placing_an_order_on_multiple_channels_with_different_currency.feature

```diff
@@ -5,8 +5,8 @@ Feature: Placing an order on multiple channels with different currency
     I want user to place orders in channel’s base currency
 
     Background:
-        Given the store operates on a channel named "United States" in "USD" currency
-        And the store operates on another channel named "Colombia" in "COP" currency
+        Given the store operates on a channel named "United States" in "USD" currency and with hostname "USA"
```

Comment:
Change these hostnames to have more sense as hostname, at least to be lowercased:
```suggestion
        Given the store operates on a channel named "United States" in "USD" currency and with hostname "usa"
```